indico/modules/events/editing/notifications.py

```diff
@@ -0,0 +1,96 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2020 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+from indico.core.notifications import email_sender, make_email
+from indico.web.flask.templating import get_template_module
+from indico.web.flask.util import url_for
+
+
+def _get_timeline_url(revision):
+    editable = revision.editable
+    contribution = editable.contribution
+    return url_for('event_editing.editable', contribution.event, contribution, type=editable.type.name, _external=True)
```

Comment:
You can pass only `contribution` since `Contribution`'s locator_property already returns event-related attributes.

```suggestion
    return url_for('event_editing.editable', contribution, type=editable.type.name, _external=True)
```

Reply:
even easier, `url_for('event_editing.editable', editable, _external=True)` since `Editable` has its own locator property.

Reply:
Yeah, even better ;)